<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Home</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/master/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>




<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Embryoid_Body_Pilot_Workflowr</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/KLRhodes/Embryoid_Body_Pilot_Workflowr">
    <span class="fa fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Home</h1>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2020-10-19
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 2 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>Embryoid_Body_Pilot_Workflowr/analysis/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.6.2). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date </a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate" class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomKLRhodesEmbryoidBodyPilotWorkflowrtree5af5a739024876486b54455ab74a0ad22c2f5c79targetblank5af5a73a"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> <a href="https://github.com/KLRhodes/Embryoid_Body_Pilot_Workflowr/tree/5af5a739024876486b54455ab74a0ad22c2f5c79" target="_blank">5af5a73</a> </a>
</p>
</div>
<div id="strongRepositoryversionstrongahrefhttpsgithubcomKLRhodesEmbryoidBodyPilotWorkflowrtree5af5a739024876486b54455ab74a0ad22c2f5c79targetblank5af5a73a" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility.
</p>
<p>
The results in this page were generated with repository version <a href="https://github.com/KLRhodes/Embryoid_Body_Pilot_Workflowr/tree/5af5a739024876486b54455ab74a0ad22c2f5c79" target="_blank">5af5a73</a>. See the <em>Past versions</em> tab to see a history of the changes made to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    analysis/.Rhistory
    Ignored:    output/.Rhistory

Untracked files:
    Untracked:  analysis/SingleCell_HierarchicalClustering_NoGeneFilter.Rmd
    Untracked:  analysis/child/
    Untracked:  code/ConvertToDGE.Rmd
    Untracked:  code/ConvertToDGE_PseudoBulk.Rmd
    Untracked:  code/EB.getHumanMetadata.Rmd
    Untracked:  code/get_genelist_byPCTthresh.Rmd
    Untracked:  figure/
    Untracked:  output/DGELists/
    Untracked:  output/GeneLists_by_minPCT/
    Untracked:  output/mergedObjects/
    Untracked:  output/pdfs/
    Untracked:  output/sampleQCrds/

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were made to the R Markdown (<code>analysis/index.Rmd</code>) and HTML (<code>docs/index.html</code>) files. If you've configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view the files as they were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/KLRhodes/Embryoid_Body_Pilot_Workflowr/blob/5af5a739024876486b54455ab74a0ad22c2f5c79/analysis/index.Rmd" target="_blank">5af5a73</a>
</td>
<td>
KLRhodes
</td>
<td>
2020-10-19
</td>
<td>
wflow_publish(&quot;analysis/index.Rmd&quot;)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/KLRhodes/Embryoid_Body_Pilot_Workflowr/dcceb2e6e1c1cf3f30bda53be9cffd2e7539416b/docs/index.html" target="_blank">dcceb2e</a>
</td>
<td>
KLRhodes
</td>
<td>
2020-08-31
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/KLRhodes/Embryoid_Body_Pilot_Workflowr/20aa37b96c35c60aaa06228d164cae10e082b01b/docs/index.html" target="_blank">20aa37b</a>
</td>
<td>
KLRhodes
</td>
<td>
2020-08-31
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/KLRhodes/Embryoid_Body_Pilot_Workflowr/blob/e50c78b76ba71d6731d49764c5db264e78b35847/analysis/index.Rmd" target="_blank">e50c78b</a>
</td>
<td>
KLRhodes
</td>
<td>
2020-08-31
</td>
<td>
wflow_publish(&quot;analysis/index.Rmd&quot;)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/KLRhodes/Embryoid_Body_Pilot_Workflowr/63a84ab82b33e2dd5cf963b759f045f9c38a6c00/docs/index.html" target="_blank">63a84ab</a>
</td>
<td>
KLRhodes
</td>
<td>
2020-08-14
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/KLRhodes/Embryoid_Body_Pilot_Workflowr/blob/7ccb0e11a60049138d0fad3b30875bcc8fd9cf48/analysis/index.Rmd" target="_blank">7ccb0e1</a>
</td>
<td>
KLRhodes
</td>
<td>
2020-08-14
</td>
<td>
wflow_publish(&quot;analysis/index.Rmd&quot;)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/KLRhodes/Embryoid_Body_Pilot_Workflowr/c780d5b9fd0af8c4a77e0e35f000b6ee741033a0/docs/index.html" target="_blank">c780d5b</a>
</td>
<td>
KLRhodes
</td>
<td>
2020-08-14
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/KLRhodes/Embryoid_Body_Pilot_Workflowr/blob/c29c3016243ba4d6c9d033e65f27234971e09124/analysis/index.Rmd" target="_blank">c29c301</a>
</td>
<td>
KLRhodes
</td>
<td>
2020-08-14
</td>
<td>
wflow_publish(&quot;analysis/index.Rmd&quot;)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/KLRhodes/Embryoid_Body_Pilot_Workflowr/e0acf54331e49361137b927c97c6eee1ba9e082d/docs/index.html" target="_blank">e0acf54</a>
</td>
<td>
KLRhodes
</td>
<td>
2020-08-14
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/KLRhodes/Embryoid_Body_Pilot_Workflowr/blob/a7fd3eac37685fc26026e04149420a6d256258fb/analysis/index.Rmd" target="_blank">a7fd3ea</a>
</td>
<td>
KLRhodes
</td>
<td>
2020-08-14
</td>
<td>
wflow_publish(&quot;analysis/index.Rmd&quot;)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/KLRhodes/Embryoid_Body_Pilot_Workflowr/40a7b23af60018707a16a41e72522dfb3e08bdf7/docs/index.html" target="_blank">40a7b23</a>
</td>
<td>
KLRhodes
</td>
<td>
2020-08-14
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/KLRhodes/Embryoid_Body_Pilot_Workflowr/blob/f36c13db8f9005efd33f9686a959bec720d1c8fe/analysis/index.Rmd" target="_blank">f36c13d</a>
</td>
<td>
KLRhodes
</td>
<td>
2020-08-14
</td>
<td>
wflow_publish(&quot;analysis/index.Rmd&quot;)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/KLRhodes/Embryoid_Body_Pilot_Workflowr/7055bcee918a9a476352e2050750eb8be4a0fb42/docs/index.html" target="_blank">7055bce</a>
</td>
<td>
KLRhodes
</td>
<td>
2020-08-14
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/KLRhodes/Embryoid_Body_Pilot_Workflowr/blob/8fcb48adad6111cff2fb1f27d265cf482df4f542/analysis/index.Rmd" target="_blank">8fcb48a</a>
</td>
<td>
KLRhodes
</td>
<td>
2020-08-14
</td>
<td>
wflow_publish(&quot;analysis/index.Rmd&quot;)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/KLRhodes/Embryoid_Body_Pilot_Workflowr/508974d19ba6332d1fac1e6674480ab44ee4127a/docs/index.html" target="_blank">508974d</a>
</td>
<td>
KLRhodes
</td>
<td>
2020-08-14
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/KLRhodes/Embryoid_Body_Pilot_Workflowr/blob/7a220f381238e9dc99c9f029ee2289981f0fba9d/analysis/index.Rmd" target="_blank">7a220f3</a>
</td>
<td>
KLRhodes
</td>
<td>
2020-08-14
</td>
<td>
wflow_publish(&quot;analysis/index.Rmd&quot;)
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/KLRhodes/Embryoid_Body_Pilot_Workflowr/blob/d9b626c26aaf9e1a8a17150923bdb6a97be624f1/analysis/index.Rmd" target="_blank">d9b626c</a>
</td>
<td>
KLRhodes
</td>
<td>
2020-08-14
</td>
<td>
wflow_git_commit(all = TRUE)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/KLRhodes/Embryoid_Body_Pilot_Workflowr/d9b626c26aaf9e1a8a17150923bdb6a97be624f1/docs/index.html" target="_blank">d9b626c</a>
</td>
<td>
KLRhodes
</td>
<td>
2020-08-14
</td>
<td>
wflow_git_commit(all = TRUE)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/KLRhodes/Embryoid_Body_Pilot_Workflowr/218f095bdab4cf5505766e714007cf1c8f5875d0/docs/index.html" target="_blank">218f095</a>
</td>
<td>
KLRhodes
</td>
<td>
2020-08-04
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/KLRhodes/Embryoid_Body_Pilot_Workflowr/blob/87c729215122743cb45822f7d8f1014d97413af1/analysis/index.Rmd" target="_blank">87c7292</a>
</td>
<td>
KLRhodes
</td>
<td>
2020-08-04
</td>
<td>
Start workflowr project.
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<p>Welcome to my research website.</p>
<p>This site contains analyses performed on human Embryoid bodies, aggregates of spontaneously differentiating cells generated from induced pluripotent stem cells.</p>
<p>Single Cell RNA-seq data was collected using the 10x platform.</p>
<div id="preprocessing-qc-and-filtering-run-emptydrops-add-metadata-to-seurat-object-from-each-10x-lane" class="section level1">
<h1><em>Preprocessing, QC, and filtering ##</em>Run EmptyDrops, Add Metadata to Seurat object from each 10x lane</h1>
<ul>
<li>to be added soon! ##*Filtering and Normalizing each 10x lane</li>
<li><a href="SampleFilteringandSCT_Batch1Lane1.html">Batch1Lane1</a><br /></li>
<li><a href="SampleFilteringandSCT_Batch1Lane3.html">Batch1Lane3</a><br /></li>
<li><a href="SampleFilteringandSCT_Batch1Lane4.html">Batch1Lane4</a><br /></li>
<li><a href="SampleFilteringandSCT_Batch1Lane5.html">Batch1Lane5</a><br /></li>
<li><a href="SampleFilteringandSCT_Batch1Lane6.html">Batch1Lane6</a><br /></li>
<li><a href="SampleFilteringandSCT_Batch1Lane7.html">Batch1Lane7</a><br /></li>
<li><a href="SampleFilteringandSCT_Batch1Lane8.html">Batch1Lane8</a><br /></li>
<li><a href="SampleFilteringandSCT_Batch1Lane9.html">Batch1Lane9</a><br /></li>
<li><a href="SampleFilteringandSCT_Batch2Lane1.html">Batch2Lane1</a><br /></li>
<li><a href="SampleFilteringandSCT_Batch2Lane2.html">Batch2Lane2</a><br /></li>
<li><a href="SampleFilteringandSCT_Batch2Lane3.html">Batch2Lane3</a><br /></li>
<li><a href="SampleFilteringandSCT_Batch2Lane4.html">Batch2Lane4</a><br /></li>
<li><a href="SampleFilteringandSCT_Batch3Lane1.html">Batch3Lane1</a><br /></li>
<li><a href="SampleFilteringandSCT_Batch3Lane2.html">Batch3Lane2</a><br /></li>
<li><a href="SampleFilteringandSCT_Batch3Lane3.html">Batch3Lane3</a><br /></li>
<li><a href="SampleFilteringandSCT_Batch3Lane4.html">Batch3Lane4</a></li>
</ul>
</div>
<div id="merging-and-integration-seurat-clustering-harmony-correcting-for-batch-harmony-correcting-for-batch-individual-harmony-correcting-for-batch-individual-and-lane-sctransform-regressing-out-batchindividual-sctransform-regressing-out-batchindividual-then-harmony-correcting-batch-and-individual-no-integration-just-merged" class="section level1">
<h1>*Merging and integration, seurat clustering + <a href="IntegrateAnalysis.afterFilter.HarmonyBatch.html">Harmony correcting for Batch</a> + <a href="IntegrateAnalysis.afterFilter.HarmonyBatchindividual.html">Harmony correcting for Batch, Individual</a><br />+ <a href="IntegrateAnalysis.afterFilter.HarmonyBatchSampleIDindividual.html">Harmony correcting for Batch, Individual, and Lane</a><br />+ <a href="IntegrateAnalysis.afterFilter.SCTregressBatchIndividual.html">SCTransform regressing out Batch,individual</a><br />+ <a href="IntegrateAnalysis.afterFilter.SCTregressBatchIndividualHarmonyBatchindividual.html">SCTransform regressing out Batch,individual then Harmony correcting batch and individual</a><br />+ <a href="IntegrateAnalysis.afterFilter.NOHARMONYjustmerge.html">No Integration, Just Merged</a></h1>
<p>Note: all analyses below use data corrected using Haromy for Batch and Individual</p>
</div>
<div id="differential-expression-with-limma-pseudobulk-using-cluster-resolution-0.1-using-cluster-resolution-0.5" class="section level1">
<h1>*Differential Expression with limma (Pseudobulk) + <a href="Pseudobulk_Limma_Harmony.BatchIndividual_ClusterRes0.1_minPCT0.2.Rmd">Using Cluster Resolution 0.1</a> + <a href="Pseudobulk_Limma_Harmony.BatchIndividual_ClusterRes0.5_minPCT0.2.Rmd">Using Cluster Resolution 0.5</a></h1>
</div>
<div id="seurat-subclustering-to-be-added-soon" class="section level1">
<h1>*Seurat subclustering + To be added soon!</h1>
</div>
<div id="cell-type-identification-using-schcl-schcl-analysis" class="section level1">
<h1>*Cell type identification using scHCL + <a href="RunscHCL_HarmonyBatchInd.Rmd">scHCL analysis</a></h1>
</div>
<div id="topic-modelling-with-fasttopics-to-be-added-soon" class="section level1">
<h1>*Topic Modelling with FastTopics + to be added soon!</h1>
</div>
<div id="hierachical-clustering-of-gene-expression-single-cell-resolution-pseudobulk-using-cluster-resolution-0.1-pseudobulk-using-cluster-resolution-0.5-pseudobulk-using-cluster-resolution-0.8-pseudobulk-using-cluster-resolution-1" class="section level1">
<h1>*Hierachical clustering of gene expression + <a href="SingleCell_HierarchicalClustering_NoGeneFilter.Rmd">Single Cell Resolution</a> + <a href="Pseudobulk_HierarchicalClustering_Harmony.Batchindividual_ClusterRes0.1_minPCT0.2.Rmd">Pseudobulk, Using Cluster Resolution 0.1</a> + <a href="Pseudobulk_HierarchicalClustering_Harmony.Batchindividual_ClusterRes0.5_minPCT0.2.Rmd">Pseudobulk, Using Cluster Resolution 0.5</a> + <a href="Pseudobulk_HierarchicalClustering_Harmony.Batchindividual_ClusterRes0.8_minPCT0.2.Rmd">Pseudobulk, Using Cluster Resolution 0.8</a> + <a href="Pseudobulk_HierarchicalClustering_Harmony.Batchindividual_ClusterRes1_minPCT0.2.Rmd">Pseudobulk, Using Cluster Resolution 1</a></h1>
</div>
<div id="variance-partition-to-determine-the-relative-contribution-of-cluster-batch-and-individual-to-variation-in-gene-expression-pseudobulk-using-cluster-resolution-0.1-pseudobulk-using-cluster-resolution-0.5-pseudobulk-using-cluster-resolution-0.8-pseudobulk-using-cluster-resolution-1" class="section level1">
<h1>*Variance Partition (to determine the relative contribution of Cluster, Batch, and Individual to variation in gene expression) + <a href="Pseudobulk_VariancePartition_Harmony.Batchindividual_ClusterRes0.1_minPCT0.2.Rmd">Pseudobulk, Using Cluster Resolution 0.1</a> + <a href="Pseudobulk_VariancePartition_Harmony.Batchindividual_ClusterRes0.5_minPCT0.2.Rmd">Pseudobulk, Using Cluster Resolution 0.5</a> + <a href="Pseudobulk_VariancePartition_Harmony.Batchindividual_ClusterRes0.8_minPCT0.2.Rmd">Pseudobulk, Using Cluster Resolution 0.8</a> + <a href="Pseudobulk_VariancePartition_Harmony.Batchindividual_ClusterRes1_minPCT0.2.Rmd">Pseudobulk, Using Cluster Resolution 1</a></h1>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
