<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Home</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/master/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>




<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Embryoid_Body_Pilot_Workflowr</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/KLRhodes/Embryoid_Body_Pilot_Workflowr">
    <span class="fa fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Home</h1>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2021-07-05
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 2 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>Embryoid_Body_Pilot_Workflowr/analysis/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.6.2). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date </a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate" class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomKLRhodesEmbryoidBodyPilotWorkflowrtree48050346f9314497fc3b1b824fbf969fd3cd7ec1targetblank4805034a"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> <a href="https://github.com/KLRhodes/Embryoid_Body_Pilot_Workflowr/tree/48050346f9314497fc3b1b824fbf969fd3cd7ec1" target="_blank">4805034</a> </a>
</p>
</div>
<div id="strongRepositoryversionstrongahrefhttpsgithubcomKLRhodesEmbryoidBodyPilotWorkflowrtree48050346f9314497fc3b1b824fbf969fd3cd7ec1targetblank4805034a" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility.
</p>
<p>
The results in this page were generated with repository version <a href="https://github.com/KLRhodes/Embryoid_Body_Pilot_Workflowr/tree/48050346f9314497fc3b1b824fbf969fd3cd7ec1" target="_blank">4805034</a>. See the <em>Past versions</em> tab to see a history of the changes made to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    analysis/.Rhistory
    Ignored:    output/.Rhistory

Untracked files:
    Untracked:  GSE122380_raw_counts.txt.gz
    Untracked:  UTF1_plots.Rmd
    Untracked:  analysis/OLD/
    Untracked:  analysis/child/
    Untracked:  build_refint_scale.R
    Untracked:  build_refint_sct.R
    Untracked:  build_stuff.R
    Untracked:  build_varpart_sc.R
    Untracked:  code/.ipynb_checkpoints/
    Untracked:  code/CellRangerPreprocess.Rmd
    Untracked:  code/GEO_processed_data.Rmd
    Untracked:  code/PowerAnalysis_NoiseRatio.ipynb
    Untracked:  code/Untitled.ipynb
    Untracked:  code/Untitled1.ipynb
    Untracked:  data/HCL_Fig1_adata.h5ad
    Untracked:  data/HCL_Fig1_adata.h5seurat
    Untracked:  data/dge/
    Untracked:  data/dge_raw_data.tar.gz
    Untracked:  data/ref.expr.rda
    Untracked:  figure/
    Untracked:  output/CR_sampleQCrds/
    Untracked:  output/CaoEtAl.Obj.CellsOfAllClusters.ProteinCodingGenes.rds
    Untracked:  output/CaoEtAl.Obj.rds
    Untracked:  output/ClusterInfo_res0.1.csv
    Untracked:  output/DGELists/
    Untracked:  output/DownSampleVarPart.rds
    Untracked:  output/Frequency.MostCommonAnnotation.FiveNearestRefCells.csv
    Untracked:  output/GEOsubmissionProcessedFiles/
    Untracked:  output/GeneLists_by_minPCT/
    Untracked:  output/MostCommonAnnotation.FiveNearestRefCells.csv
    Untracked:  output/NearestReferenceCell.Cao.hESC.EuclideanDistanceinHarmonySpace.csv
    Untracked:  output/NearestReferenceCell.Cao.hESC.FrequencyofEachAnnotation.csv
    Untracked:  output/NearestReferenceCell.SCTregressRNAassay.Cao.hESC.EuclideanDistanceinHarmonySpace.csv
    Untracked:  output/NearestReferenceCell.SCTregressRNAassay.Cao.hESC.FrequencyofEachAnnotation.csv
    Untracked:  output/Pseudobulk_Limma_res0.1_OnevAllTopTables.csv
    Untracked:  output/Pseudobulk_Limma_res0.1_OnevAll_top10Upregby_adjP.csv
    Untracked:  output/Pseudobulk_Limma_res0.1_OnevAll_top10Upregby_logFC.csv
    Untracked:  output/Pseudobulk_Limma_res0.5_OnevAllTopTables.csv
    Untracked:  output/Pseudobulk_Limma_res0.8_OnevAllTopTables.csv
    Untracked:  output/Pseudobulk_Limma_res1_OnevAllTopTables.csv
    Untracked:  output/Pseudobulk_VarPart.ByCluster.Res0.1.rds
    Untracked:  output/ResidualVariances_fromDownSampAnalysis.csv
    Untracked:  output/SingleCell_VariancePartition_RNA_Res0.1_minPCT0.2.rds
    Untracked:  output/SingleCell_VariancePartition_Res0.1_minPCT0.2.rds
    Untracked:  output/SingleCell_VariancePartition_SCT_Res0.1_minPCT0.2.rds
    Untracked:  output/TopicModelling_k10_top10drivergenes.byBeta.csv
    Untracked:  output/TopicModelling_k6_top10drivergenes.byBeta.csv
    Untracked:  output/TopicModelling_k6_top15drivergenes.byZ.csv
    Untracked:  output/TranferredAnnotations_ReferenceInt_JustEarlyEcto.csv
    Untracked:  output/TranferredAnnotations_ReferenceInt_JustEndoderm.csv
    Untracked:  output/TranferredAnnotations_ReferenceInt_JustMeso.csv
    Untracked:  output/TranferredAnnotations_ReferenceInt_JustNeuralCrest.csv
    Untracked:  output/TranferredAnnotations_ReferenceInt_JustNeuron.csv
    Untracked:  output/TranferredAnnotations_ReferenceInt_JustPluripotent.csv
    Untracked:  output/VarPart.ByCluster.Res0.1.rds
    Untracked:  output/azimuth/
    Untracked:  output/downsamp_10800cells_10subreps_medianexplainedbyresiduals_varpart_PsB.rds
    Untracked:  output/downsamp_16200cells_10subreps_medianexplainedbyresiduals_varpart_PsB.rds
    Untracked:  output/downsamp_21600cells_10subreps_medianexplainedbyresiduals_varpart_PsB.rds
    Untracked:  output/downsamp_2700cells_10subreps_medianexplainedbyresiduals_varpart_PsB.rds
    Untracked:  output/downsamp_2700cells_10subreps_medianexplainedbyresiduals_varpart_scres.rds
    Untracked:  output/downsamp_5400cells_10subreps_medianexplainedbyresiduals_varpart_PsB.rds
    Untracked:  output/downsamp_7200cells_10subreps_medianexplainedbyresiduals_varpart_PsB.rds
    Untracked:  output/fasttopics/
    Untracked:  output/figs/
    Untracked:  output/merge.Cao.SCTwRegressOrigIdent.rds
    Untracked:  output/merge.all.SCTwRegressOrigIdent.Harmony.rds
    Untracked:  output/merged.SCT.counts.matrix.rds
    Untracked:  output/merged.raw.counts.matrix.rds
    Untracked:  output/mergedObjects/
    Untracked:  output/pdfs/
    Untracked:  output/sampleQCrds/
    Untracked:  output/splitgpm_gsea_results/
    Untracked:  slurm-12005914.out
    Untracked:  slurm-12005923.out

Unstaged changes:
    Deleted:    analysis/IntegrateAnalysis.afterFilter.HarmonyBatch.Rmd
    Deleted:    analysis/IntegrateAnalysis.afterFilter.HarmonyBatchSampleIDindividual.Rmd
    Deleted:    analysis/IntegrateAnalysis.afterFilter.NOHARMONYjustmerge.Rmd
    Deleted:    analysis/IntegrateAnalysis.afterFilter.SCTregressBatchIndividual.Rmd
    Deleted:    analysis/IntegrateAnalysis.afterFilter.SCTregressBatchIndividualHarmonyBatchindividual.Rmd
    Deleted:    analysis/RunscHCL_HarmonyBatchInd.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were made to the R Markdown (<code>analysis/index.Rmd</code>) and HTML (<code>docs/index.html</code>) files. If you've configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view the files as they were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/KLRhodes/Embryoid_Body_Pilot_Workflowr/blob/48050346f9314497fc3b1b824fbf969fd3cd7ec1/analysis/index.Rmd" target="_blank">4805034</a>
</td>
<td>
KLRhodes
</td>
<td>
2021-07-05
</td>
<td>
wflow_publish(&quot;analysis/index.Rmd&quot;)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/KLRhodes/Embryoid_Body_Pilot_Workflowr/be3f7660652a93e5dc6cc01cf3553ed26c782ea4/docs/index.html" target="_blank">be3f766</a>
</td>
<td>
KLRhodes
</td>
<td>
2021-07-05
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/KLRhodes/Embryoid_Body_Pilot_Workflowr/blob/8c34ca9fb9703193cee0ec8f207331e837187c7f/analysis/index.Rmd" target="_blank">8c34ca9</a>
</td>
<td>
KLRhodes
</td>
<td>
2021-07-05
</td>
<td>
wflow_publish(&quot;analysis/index.Rmd&quot;)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/KLRhodes/Embryoid_Body_Pilot_Workflowr/ba90a384f7031c7f1c603b0f52f12138e100b199/docs/index.html" target="_blank">ba90a38</a>
</td>
<td>
KLRhodes
</td>
<td>
2021-07-05
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/KLRhodes/Embryoid_Body_Pilot_Workflowr/blob/4123658a15772363aabe7e252b26729c15cd1419/analysis/index.Rmd" target="_blank">4123658</a>
</td>
<td>
KLRhodes
</td>
<td>
2021-07-05
</td>
<td>
wflow_publish(&quot;analysis/index.Rmd&quot;)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/KLRhodes/Embryoid_Body_Pilot_Workflowr/69e282420d9611ad3d7170b416d374163ba1f30d/docs/index.html" target="_blank">69e2824</a>
</td>
<td>
KLRhodes
</td>
<td>
2021-07-05
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/KLRhodes/Embryoid_Body_Pilot_Workflowr/blob/74c29149084d2012f3347e028c48d48283b8e075/analysis/index.Rmd" target="_blank">74c2914</a>
</td>
<td>
KLRhodes
</td>
<td>
2021-07-05
</td>
<td>
wflow_publish(&quot;analysis/index.Rmd&quot;)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/KLRhodes/Embryoid_Body_Pilot_Workflowr/49d5ea0fb03b35da2eedbd70cd670ee7ec2e757a/docs/index.html" target="_blank">49d5ea0</a>
</td>
<td>
KLRhodes
</td>
<td>
2020-10-19
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/KLRhodes/Embryoid_Body_Pilot_Workflowr/blob/1638e40588f12d28ecdb3564584e648b5a777483/analysis/index.Rmd" target="_blank">1638e40</a>
</td>
<td>
KLRhodes
</td>
<td>
2020-10-19
</td>
<td>
wflow_publish(&quot;analysis/index.Rmd&quot;)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/KLRhodes/Embryoid_Body_Pilot_Workflowr/d1ee77ee9fba9cb5f77b634600346b1bbb0155bc/docs/index.html" target="_blank">d1ee77e</a>
</td>
<td>
KLRhodes
</td>
<td>
2020-10-19
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/KLRhodes/Embryoid_Body_Pilot_Workflowr/blob/747f854bded1c1d897da82886ff5f8a188f05add/analysis/index.Rmd" target="_blank">747f854</a>
</td>
<td>
KLRhodes
</td>
<td>
2020-10-19
</td>
<td>
wflow_publish(&quot;analysis/index.Rmd&quot;)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/KLRhodes/Embryoid_Body_Pilot_Workflowr/667319682b43de85a4ea245dccc9d375150006c7/docs/index.html" target="_blank">6673196</a>
</td>
<td>
KLRhodes
</td>
<td>
2020-10-19
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/KLRhodes/Embryoid_Body_Pilot_Workflowr/blob/c2b8b194cacfd7914f1d222a7f1f8a75efe10f58/analysis/index.Rmd" target="_blank">c2b8b19</a>
</td>
<td>
KLRhodes
</td>
<td>
2020-10-19
</td>
<td>
wflow_publish(&quot;analysis/index.Rmd&quot;)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/KLRhodes/Embryoid_Body_Pilot_Workflowr/a75233d74f9c4c87c20036ac9c1aa48136677657/docs/index.html" target="_blank">a75233d</a>
</td>
<td>
KLRhodes
</td>
<td>
2020-10-19
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/KLRhodes/Embryoid_Body_Pilot_Workflowr/blob/3b424e0ebabae11e38430ee3bf7eb6383d35f10d/analysis/index.Rmd" target="_blank">3b424e0</a>
</td>
<td>
KLRhodes
</td>
<td>
2020-10-19
</td>
<td>
wflow_publish(&quot;analysis/index.Rmd&quot;)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/KLRhodes/Embryoid_Body_Pilot_Workflowr/06d1fce79a0381b45dbda5c6801f4ec74480650f/docs/index.html" target="_blank">06d1fce</a>
</td>
<td>
KLRhodes
</td>
<td>
2020-10-19
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/KLRhodes/Embryoid_Body_Pilot_Workflowr/blob/1abb092b58cfb555ed0e84dc516395fcf76e77e6/analysis/index.Rmd" target="_blank">1abb092</a>
</td>
<td>
KLRhodes
</td>
<td>
2020-10-19
</td>
<td>
wflow_publish(&quot;analysis/index.Rmd&quot;)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/KLRhodes/Embryoid_Body_Pilot_Workflowr/bae15ab170c53056f960465aed4085f3148af55d/docs/index.html" target="_blank">bae15ab</a>
</td>
<td>
KLRhodes
</td>
<td>
2020-10-19
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/KLRhodes/Embryoid_Body_Pilot_Workflowr/blob/c4d03aa1fba373bf74ae1692767f7f49b6aa86a4/analysis/index.Rmd" target="_blank">c4d03aa</a>
</td>
<td>
KLRhodes
</td>
<td>
2020-10-19
</td>
<td>
wflow_publish(&quot;analysis/index.Rmd&quot;)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/KLRhodes/Embryoid_Body_Pilot_Workflowr/968ead01991e486d0bcc9caf6313ecc7eb9d358e/docs/index.html" target="_blank">968ead0</a>
</td>
<td>
KLRhodes
</td>
<td>
2020-10-19
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/KLRhodes/Embryoid_Body_Pilot_Workflowr/blob/77fd6c38a67705262e373c61f1b7a1594f964daf/analysis/index.Rmd" target="_blank">77fd6c3</a>
</td>
<td>
KLRhodes
</td>
<td>
2020-10-19
</td>
<td>
wflow_publish(&quot;analysis/index.Rmd&quot;)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/KLRhodes/Embryoid_Body_Pilot_Workflowr/51b43d361a4099a7941e6635592ae45579894203/docs/index.html" target="_blank">51b43d3</a>
</td>
<td>
KLRhodes
</td>
<td>
2020-10-19
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/KLRhodes/Embryoid_Body_Pilot_Workflowr/blob/2fc2a642144e9dc217d98974e8d9eae2d67275b1/analysis/index.Rmd" target="_blank">2fc2a64</a>
</td>
<td>
KLRhodes
</td>
<td>
2020-10-19
</td>
<td>
wflow_publish(&quot;analysis/index.Rmd&quot;)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/KLRhodes/Embryoid_Body_Pilot_Workflowr/40eb61e26a349e2f08c094d067597f58fe8fd864/docs/index.html" target="_blank">40eb61e</a>
</td>
<td>
KLRhodes
</td>
<td>
2020-10-19
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/KLRhodes/Embryoid_Body_Pilot_Workflowr/blob/5af5a739024876486b54455ab74a0ad22c2f5c79/analysis/index.Rmd" target="_blank">5af5a73</a>
</td>
<td>
KLRhodes
</td>
<td>
2020-10-19
</td>
<td>
wflow_publish(&quot;analysis/index.Rmd&quot;)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/KLRhodes/Embryoid_Body_Pilot_Workflowr/dcceb2e6e1c1cf3f30bda53be9cffd2e7539416b/docs/index.html" target="_blank">dcceb2e</a>
</td>
<td>
KLRhodes
</td>
<td>
2020-08-31
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/KLRhodes/Embryoid_Body_Pilot_Workflowr/20aa37b96c35c60aaa06228d164cae10e082b01b/docs/index.html" target="_blank">20aa37b</a>
</td>
<td>
KLRhodes
</td>
<td>
2020-08-31
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/KLRhodes/Embryoid_Body_Pilot_Workflowr/blob/e50c78b76ba71d6731d49764c5db264e78b35847/analysis/index.Rmd" target="_blank">e50c78b</a>
</td>
<td>
KLRhodes
</td>
<td>
2020-08-31
</td>
<td>
wflow_publish(&quot;analysis/index.Rmd&quot;)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/KLRhodes/Embryoid_Body_Pilot_Workflowr/63a84ab82b33e2dd5cf963b759f045f9c38a6c00/docs/index.html" target="_blank">63a84ab</a>
</td>
<td>
KLRhodes
</td>
<td>
2020-08-14
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/KLRhodes/Embryoid_Body_Pilot_Workflowr/blob/7ccb0e11a60049138d0fad3b30875bcc8fd9cf48/analysis/index.Rmd" target="_blank">7ccb0e1</a>
</td>
<td>
KLRhodes
</td>
<td>
2020-08-14
</td>
<td>
wflow_publish(&quot;analysis/index.Rmd&quot;)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/KLRhodes/Embryoid_Body_Pilot_Workflowr/c780d5b9fd0af8c4a77e0e35f000b6ee741033a0/docs/index.html" target="_blank">c780d5b</a>
</td>
<td>
KLRhodes
</td>
<td>
2020-08-14
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/KLRhodes/Embryoid_Body_Pilot_Workflowr/blob/c29c3016243ba4d6c9d033e65f27234971e09124/analysis/index.Rmd" target="_blank">c29c301</a>
</td>
<td>
KLRhodes
</td>
<td>
2020-08-14
</td>
<td>
wflow_publish(&quot;analysis/index.Rmd&quot;)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/KLRhodes/Embryoid_Body_Pilot_Workflowr/e0acf54331e49361137b927c97c6eee1ba9e082d/docs/index.html" target="_blank">e0acf54</a>
</td>
<td>
KLRhodes
</td>
<td>
2020-08-14
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/KLRhodes/Embryoid_Body_Pilot_Workflowr/blob/a7fd3eac37685fc26026e04149420a6d256258fb/analysis/index.Rmd" target="_blank">a7fd3ea</a>
</td>
<td>
KLRhodes
</td>
<td>
2020-08-14
</td>
<td>
wflow_publish(&quot;analysis/index.Rmd&quot;)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/KLRhodes/Embryoid_Body_Pilot_Workflowr/40a7b23af60018707a16a41e72522dfb3e08bdf7/docs/index.html" target="_blank">40a7b23</a>
</td>
<td>
KLRhodes
</td>
<td>
2020-08-14
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/KLRhodes/Embryoid_Body_Pilot_Workflowr/blob/f36c13db8f9005efd33f9686a959bec720d1c8fe/analysis/index.Rmd" target="_blank">f36c13d</a>
</td>
<td>
KLRhodes
</td>
<td>
2020-08-14
</td>
<td>
wflow_publish(&quot;analysis/index.Rmd&quot;)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/KLRhodes/Embryoid_Body_Pilot_Workflowr/7055bcee918a9a476352e2050750eb8be4a0fb42/docs/index.html" target="_blank">7055bce</a>
</td>
<td>
KLRhodes
</td>
<td>
2020-08-14
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/KLRhodes/Embryoid_Body_Pilot_Workflowr/blob/8fcb48adad6111cff2fb1f27d265cf482df4f542/analysis/index.Rmd" target="_blank">8fcb48a</a>
</td>
<td>
KLRhodes
</td>
<td>
2020-08-14
</td>
<td>
wflow_publish(&quot;analysis/index.Rmd&quot;)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/KLRhodes/Embryoid_Body_Pilot_Workflowr/508974d19ba6332d1fac1e6674480ab44ee4127a/docs/index.html" target="_blank">508974d</a>
</td>
<td>
KLRhodes
</td>
<td>
2020-08-14
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/KLRhodes/Embryoid_Body_Pilot_Workflowr/blob/7a220f381238e9dc99c9f029ee2289981f0fba9d/analysis/index.Rmd" target="_blank">7a220f3</a>
</td>
<td>
KLRhodes
</td>
<td>
2020-08-14
</td>
<td>
wflow_publish(&quot;analysis/index.Rmd&quot;)
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/KLRhodes/Embryoid_Body_Pilot_Workflowr/blob/d9b626c26aaf9e1a8a17150923bdb6a97be624f1/analysis/index.Rmd" target="_blank">d9b626c</a>
</td>
<td>
KLRhodes
</td>
<td>
2020-08-14
</td>
<td>
wflow_git_commit(all = TRUE)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/KLRhodes/Embryoid_Body_Pilot_Workflowr/d9b626c26aaf9e1a8a17150923bdb6a97be624f1/docs/index.html" target="_blank">d9b626c</a>
</td>
<td>
KLRhodes
</td>
<td>
2020-08-14
</td>
<td>
wflow_git_commit(all = TRUE)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/KLRhodes/Embryoid_Body_Pilot_Workflowr/218f095bdab4cf5505766e714007cf1c8f5875d0/docs/index.html" target="_blank">218f095</a>
</td>
<td>
KLRhodes
</td>
<td>
2020-08-04
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/KLRhodes/Embryoid_Body_Pilot_Workflowr/blob/87c729215122743cb45822f7d8f1014d97413af1/analysis/index.Rmd" target="_blank">87c7292</a>
</td>
<td>
KLRhodes
</td>
<td>
2020-08-04
</td>
<td>
Start workflowr project.
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<p>Welcome! This is a research website detailing analyses analyses performed on scRNA-seq data from human embryoid bodies, aggregates of spontaneously differentiating cells generated from induced pluripotent stem cells.</p>
<p>Preprint is now available on BioRxiv: <a href="https://www.biorxiv.org/content/10.1101/2021.06.16.448714v1" class="uri">https://www.biorxiv.org/content/10.1101/2021.06.16.448714v1</a></p>
<p>The overarching goals of this work are to:</p>
<ol style="list-style-type: decimal">
<li>characterize tthe diversity of cell types resulting from this particular differentiation protocol</li>
<li>Understand the contribution of biological and technical variation present in all levels of this this type of data i)What is the relative contribution of individual and replicate to variation in cell composition?</li>
</ol>
<ol start="2" style="list-style-type: lower-roman">
<li>What is the relative contribution of individual and replicate to variation in gene expression?</li>
<li>What is the relative contribution of individual and replicate to variation in gene expression dynamics across distinct developmental trajectories?</li>
</ol>
<ol start="3" style="list-style-type: decimal">
<li>Evaluate the future prospects of this model system (EBs + single cell) for identification of dynamic QTLs</li>
</ol>
<p>Study Design:</p>
<p>Embryoid bodies were formed from 3 human iPSCs lines in 3 replicates. Three weeks after formation, EBs were dissociated and scRNA-seq data was collected using the 10x platform. After quality control and filtering, this dataset consists of 42,488 cells.</p>
<div id="alignment-and-preprocessing" class="section level1">
<h1>Alignment and Preprocessing</h1>
<p>See full alignment and preprocessing pipeline here: <a href="https://github.com/kennethabarr/HumanChimp" class="uri">https://github.com/kennethabarr/HumanChimp</a></p>
</div>
<div id="run-emptydrops-add-metadata-to-seurat-object-from-each-10x-lane" class="section level1">
<h1>Run EmptyDrops, Add Metadata to Seurat object from each 10x lane</h1>
<p>See code directory: <a href="https://github.com/KLRhodes/Embryoid_Body_Pilot_Workflowr/blob/master/code/EB.getHumanMetadata.Rmd" class="uri">https://github.com/KLRhodes/Embryoid_Body_Pilot_Workflowr/blob/master/code/EB.getHumanMetadata.Rmd</a></p>
</div>
<div id="filtering-and-normalizing-each-10x-lane" class="section level1">
<h1>Filtering and Normalizing each 10x lane</h1>
<ul>
<li><a href="SampleFilteringandSCT_Batch1Lane1.html">Batch1Lane1</a><br /></li>
<li><a href="SampleFilteringandSCT_Batch1Lane3.html">Batch1Lane3</a><br /></li>
<li><a href="SampleFilteringandSCT_Batch1Lane4.html">Batch1Lane4</a><br /></li>
<li><a href="SampleFilteringandSCT_Batch1Lane5.html">Batch1Lane5</a><br /></li>
<li><a href="SampleFilteringandSCT_Batch1Lane6.html">Batch1Lane6</a><br /></li>
<li><a href="SampleFilteringandSCT_Batch1Lane7.html">Batch1Lane7</a><br /></li>
<li><a href="SampleFilteringandSCT_Batch1Lane8.html">Batch1Lane8</a><br /></li>
<li><a href="SampleFilteringandSCT_Batch1Lane9.html">Batch1Lane9</a><br /></li>
<li><a href="SampleFilteringandSCT_Batch2Lane1.html">Batch2Lane1</a><br /></li>
<li><a href="SampleFilteringandSCT_Batch2Lane2.html">Batch2Lane2</a><br /></li>
<li><a href="SampleFilteringandSCT_Batch2Lane3.html">Batch2Lane3</a><br /></li>
<li><a href="SampleFilteringandSCT_Batch2Lane4.html">Batch2Lane4</a><br /></li>
<li><a href="SampleFilteringandSCT_Batch3Lane1.html">Batch3Lane1</a><br /></li>
<li><a href="SampleFilteringandSCT_Batch3Lane2.html">Batch3Lane2</a><br /></li>
<li><a href="SampleFilteringandSCT_Batch3Lane3.html">Batch3Lane3</a><br /></li>
<li><a href="SampleFilteringandSCT_Batch3Lane4.html">Batch3Lane4</a></li>
</ul>
</div>
<div id="merging-and-integration-seurat-clustering" class="section level1">
<h1>Merging and integration, seurat clustering</h1>
<ul>
<li><a href="IntegrateAnalysis.afterFilter.HarmonyBatchindividual.html">Harmony correcting for Batch, Individual</a></li>
</ul>
</div>
<div id="differential-expression-with-limma-pseudobulk" class="section level1">
<h1>Differential Expression with limma (Pseudobulk)</h1>
<ul>
<li><a href="Pseudobulk_Limma_Harmony.BatchIndividual_ClusterRes0.1_minPCT0.2.html">Using Cluster Resolution 0.1</a></li>
<li><a href="Pseudobulk_Limma_Harmony.BatchIndividual_ClusterRes0.5_minPCT0.2.html">Using Cluster Resolution 0.5</a></li>
<li><a href="Pseudobulk_Limma_Harmony.BatchIndividual_ClusterRes0.8_minPCT0.2.html">Using Cluster Resolution 0.8</a></li>
<li><a href="Pseudobulk_Limma_Harmony.BatchIndividual_ClusterRes1_minPCT0.2.html">Using Cluster Resolution 1</a></li>
</ul>
</div>
<div id="reference-integration" class="section level1">
<h1>Reference Integration</h1>
<p>Here, we integrate our EB data with 1) scRNA-seq data from human fetal tissues (Cao et al. 2020) and with Embryoid body cells and human embryonic stem cells from Human Cell Landscape (Han et al. 2020)</p>
<ul>
<li><a href="IntegrateReference_SCTregressCaoPlusScHCL.Rmd">Integrate Reference and Annotate EB cells</a></li>
</ul>
<p>We then checked that the integration and annotation procedure was robust by subsetting our EB data to only cells for a particular type and re-integrated to check that most cells were annotated the same as in the full integration.</p>
<ul>
<li><a href="IntegrateReference_SCTregressCaoPlusScHCL_JustEarlyEcto.html">Integrate Reference w/ just Early Ectoderm EB cells</a></li>
<li><a href="IntegrateReference_SCTregressCaoPlusScHCL_JustEndo.html">Integrate Reference w/ just Endoderm EB cells</a></li>
<li><a href="IntegrateReference_SCTregressCaoPlusScHCL_JustMeso.html">Integrate Reference w/ just Mesoderm EB cells</a></li>
<li><a href="IntegrateReference_SCTregressCaoPlusScHCL_JustNeuralCrest.html">Integrate Reference w/ just Early Neural Crest EB cells</a></li>
<li><a href="IntegrateReference_SCTregressCaoPlusScHCL_JustNeuron.html">Integrate Reference w/ just Early Neural EB cells</a></li>
<li><a href="IntegrateReference_SCTregressCaoPlusScHCL_JustPluri.html">Integrate Reference w/ just Pluripotent/undifferentiated EB cells</a></li>
<li><a href="RefInt_ComparingFulltoPartialIntegrationAnnotations.html">Check Robustness of Annotations between integrations</a></li>
<li><a href="ReferenceAnn_DE.html">Check differential expression of annotated cells</a></li>
</ul>
</div>
<div id="topic-modelling-with-fasttopics" class="section level1">
<h1>Topic Modelling with FastTopics</h1>
<p>Topic modelling Scripts can be found in the code directory.</p>
<ul>
<li><a href="k6topics_Explore.html">k=6 Topics: Figures</a></li>
<li><a href="k10topics_Explore.html">k=10 Topics: Figures</a></li>
<li><a href="CompiledFits_BatchvInd.html">Topics: Replicate vs. Individual comparison</a></li>
</ul>
</div>
<div id="hierachical-clustering-of-gene-expression" class="section level1">
<h1>Hierachical clustering of gene expression</h1>
<ul>
<li><a href="Pseudobulk_HierarchicalClustering_Harmony.Batchindividual_ClusterRes0.1_minPCT0.2.html">Pseudobulk, Using Cluster Resolution 0.1</a></li>
<li><a href="Pseudobulk_HierarchicalClustering_Harmony.Batchindividual_ClusterRes0.5_minPCT0.2.html">Pseudobulk, Using Cluster Resolution 0.5</a></li>
<li><a href="Pseudobulk_HierarchicalClustering_Harmony.Batchindividual_ClusterRes0.8_minPCT0.2.html">Pseudobulk, Using Cluster Resolution 0.8</a></li>
<li><a href="Pseudobulk_HierarchicalClustering_Harmony.Batchindividual_ClusterRes1_minPCT0.2.html">Pseudobulk, Using Cluster Resolution 1</a></li>
<li><a href="SingleCell_HierarchicalClustering_NoGeneFilter.html">Single Cell, Using Cluster Resolution 0.1 and no gene filter</a></li>
</ul>
</div>
<div id="variance-partition-to-determine-the-relative-contribution-of-cluster-batch-and-individual-to-variation-in-gene-expression" class="section level1">
<h1>Variance Partition (to determine the relative contribution of Cluster, Batch, and Individual to variation in gene expression)</h1>
<ul>
<li><a href="Pseudobulk_VariancePartition_Harmony.Batchindividual_ClusterRes0.1_minPCT0.2.html">Pseudobulk, Using Cluster Resolution 0.1</a></li>
<li><a href="Pseudobulk_VariancePartition_Harmony.Batchindividual_ClusterRes0.5_minPCT0.2.html">Pseudobulk, Using Cluster Resolution 0.5</a></li>
<li><a href="Pseudobulk_VariancePartition_Harmony.Batchindividual_ClusterRes0.8_minPCT0.2.html">Pseudobulk, Using Cluster Resolution 0.8</a></li>
<li><a href="Pseudobulk_VariancePartition_Harmony.Batchindividual_ClusterRes1_minPCT0.2.html">Pseudobulk, Using Cluster Resolution 1</a></li>
<li><a href="Pseudobulk_VariancePartition_Harmony.Batchindividual_ClusterRes0.1_byCluster.html">Pseudobulk, Using Cluster Resolution 0.1 -- BY CLUSTER</a></li>
<li>Single Cell, Cluster Resolution 0.1 -- see: VarPart_SC_res0.1_SCT.Rmd</li>
<li><a href="SingleCell_VariancePartitionByCluster_Harmony.Batchindividual_ClusterRes0.1_minPCT0.2.html">Single Cell, Cluster Resolution 0.1 -- BY CLUSTER</a></li>
<li><a href="VarPartPlots_res0.1_SCT.html">Single Cell VariancePartition clean figures</a></li>
</ul>
</div>
<div id="down-sampling-and-power-analysis" class="section level1">
<h1>Down Sampling and Power Analysis</h1>
<ul>
<li><a href="DownSamp_NoiseRatio.html">Power Analysis</a></li>
</ul>
</div>
<div id="trajectory-inference-with-scanpypaga-and-identificationexploration-of-dynamic-gene-module-with-split-gpm" class="section level1">
<h1>Trajectory Inference with Scanpy/PAGA and Identification/Exploration of Dynamic Gene Module with Split-GPM</h1>
<p>See Josh Popp's GitHub: <a href="https://github.com/jmp448/ebpilot" class="uri">https://github.com/jmp448/ebpilot</a></p>
</div>
<div id="scripts-for-generating-clean-paper-figures" class="section level1">
<h1>Scripts for generating clean paper figures</h1>
<ul>
<li><a href="Fig1.Rmd">Fig. 1</a></li>
<li><a href="Fig2.Rmd">Fig. 2</a></li>
</ul>
</div>
<div id="additional-processing-and-analyses-that-were-tested-but-ultimately-not-included-in-the-paper" class="section level1">
<h1>Additional Processing and Analyses that were tested but ultimately not included in the paper:</h1>
<p>Other integrations w/ and w/out harmony:</p>
<ul>
<li><a href="IntegrateAnalysis.afterFilter.HarmonyBatch.html">Harmony correcting for Batch</a></li>
<li><a href="IntegrateAnalysis.afterFilter.HarmonyBatchSampleIDindividual.html">Harmony correcting for Batch, Individual, and Lane</a><br /></li>
<li><a href="IntegrateAnalysis.afterFilter.NOHARMONYjustmerge.html">No Integration, Just Merged</a></li>
</ul>
<p>Cell type identification using scHCL</p>
<ul>
<li><a href="RunscHCL_HarmonyBatchInd.html">scHCL analysis</a></li>
</ul>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
