<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="klrhodes" />


<title>HC_EB_sampleQC_DoubletsAndChimpRemoved</title>

<script src="site_libs/header-attrs-2.3/header-attrs.js"></script>
<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/master/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>




<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Embryoid_Body_Pilot_Workflowr</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">HC_EB_sampleQC_DoubletsAndChimpRemoved</h1>
<h4 class="author">klrhodes</h4>
<h4 class="date">8/4/2020</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2020-08-04
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>Embryoid_Body_Pilot_Workflowr/analysis/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.6.2). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date </a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate" class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20200804code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20200804)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20200804code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20200804)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstronga7fdd37"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> a7fdd37 </a>
</p>
</div>
<div id="strongRepositoryversionstronga7fdd37" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility.
</p>
<p>
The results in this page were generated with repository version a7fdd37. See the <em>Past versions</em> tab to see a history of the changes made to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/

Untracked files:
    Untracked:  analysis/IntegrateAnalysis.afterFilter.HarmonyBatchSampleIDindividual.Rmd
    Untracked:  analysis/child/
    Untracked:  code/EB.getHumanMetadata.Rmd
    Untracked:  figure/
    Untracked:  output/sampleQCrds/

Unstaged changes:
    Modified:   .Rprofile
    Modified:   .gitattributes
    Modified:   .gitignore
    Modified:   Embryoid_Body_Pilot_Workflowr.Rproj
    Modified:   README.md
    Modified:   _workflowr.yml
    Modified:   analysis/_site.yml
    Modified:   analysis/about.Rmd
    Modified:   analysis/index.Rmd
    Modified:   analysis/license.Rmd
    Modified:   code/README.md
    Modified:   data/README.md
    Modified:   output/README.md

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were made to the R Markdown (<code>analysis/SampleFilteringandSCT_Batch3Lane3.Rmd</code>) and HTML (<code>docs/SampleFilteringandSCT_Batch3Lane3.html</code>) files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view the files as they were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
a7fdd37
</td>
<td>
KLRhodes
</td>
<td>
2020-08-04
</td>
<td>
wflow_publish(c(“analysis/SampleFilteringandSCT_Batch1Lane3.Rmd”,
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<pre class="r"><code>library(knitr)
library(Seurat)
library(Matrix)
library(DropletUtils)
library(ggplot2)
library(dplyr)
library(here)</code></pre>
<p>load rds</p>
<pre class="r"><code>samp.obj&lt;- readRDS(&#39;/project2/gilad/katie/Pilot_HumanEBs/HumanEB_QCmetadata/Batch3_Lane3_seurat_NoNorm_QCMetaAdded.rds&#39;)</code></pre>
<pre class="r"><code>qc&lt;- knitr::knit_expand(file = here::here(&quot;analysis/child/SampleFilteringAndSCT.Rmd&quot;))</code></pre>
<p>load libraries</p>
<pre class="r"><code>library(Seurat)
library(Matrix)
library(DropletUtils)
library(ggplot2)
library(dplyr)
options(future.globals.maxSize= 4000*1024^2) # allow global exceeding 4Gb</code></pre>
<p>Note: This gets run after EB.metadata.Rmd (located in code directory), which read star solo output, adds metadata for species, runs emptydrops and adds QC metadata, runs demuxlet and adds individual metadata, and %MT and %ribosomal to the metadata. This facilitates filtering done here.</p>
<pre class="r"><code>head(samp.obj)</code></pre>
<pre><code>An object of class Seurat 
1 features across 5901 samples within 1 assay 
Active assay: RNA (1 features, 0 variable features)</code></pre>
<pre class="r"><code>#remove doublets (and chimp, which are assigned to doublet in this case)
samp.obj&lt;- subset(samp.obj, subset = individual != &#39;Doublet&#39;)

#remove cells assigned to individuals not actually in this sample. presumably these are low quality cells that could not be accurately assigned, so their closest match was another individual in the vcf file.
samp.obj&lt;- subset(samp.obj, subset = individual != &#39;SNG-SCM-12.variant&#39;)
samp.obj&lt;- subset(samp.obj, subset = individual != &#39;SNG-SCM-13.variant&#39;)
samp.obj&lt;- subset(samp.obj, subset = individual != &#39;SNG-SCM-6.variant&#39;)
samp.obj&lt;- subset(samp.obj, subset = individual != &#39;SNG-SCM-7.variant&#39;)
samp.obj&lt;- subset(samp.obj, subset = individual != &#39;SNG-SCM-8.variant&#39;)

#remove non-YRI human individuals (non-YRI humans only included in 1 replicate, so not needed for exploration of bio and tech variance)
samp.obj&lt;- subset(samp.obj, subset = individual != &#39;SNG-SCM-10.variant&#39;)
samp.obj&lt;- subset(samp.obj, subset = individual != &#39;SNG-SCM-9.variant&#39;)</code></pre>
<pre class="r"><code>head(samp.obj)</code></pre>
<pre><code>An object of class Seurat 
1 features across 5582 samples within 1 assay 
Active assay: RNA (1 features, 0 variable features)</code></pre>
<pre class="r"><code>samp.obj&lt;- subset(samp.obj, subset = EmptyDrops.FDR &lt; .0001) # very strict, may lose some &quot;good&quot; cells, but limits set to high quality cells and there are enough cells remaining for downstream analyses</code></pre>
<pre class="r"><code>head(samp.obj)</code></pre>
<pre><code>An object of class Seurat 
1 features across 4757 samples within 1 assay 
Active assay: RNA (1 features, 0 variable features)</code></pre>
<pre class="r"><code>VlnPlot(samp.obj, features= &quot;percent.mt&quot;, group.by = &quot;individual&quot;, pt.size = 0)</code></pre>
<p><img src="figure/SampleFilteringandSCT_Batch3Lane3.Rmd/unnamed-chunk-10-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>#visualize feature-feature relationships with FeatureScatter
plot1&lt;- FeatureScatter(samp.obj, feature1 = &quot;nCount_RNA&quot;, feature2 = &quot;percent.mt&quot;, group.by = &quot;individual&quot;)
plot2&lt;- FeatureScatter(samp.obj, feature1 = &quot;nCount_RNA&quot;, feature2 = &quot;nFeature_RNA&quot;, group.by = &quot;individual&quot;)
CombinePlots(plots= list(plot1, plot2), legend = &quot;right&quot;)</code></pre>
<pre><code>Warning: CombinePlots is being deprecated. Plots should now be combined using
the patchwork system.</code></pre>
<p><img src="figure/SampleFilteringandSCT_Batch3Lane3.Rmd/unnamed-chunk-11-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>#filtering out cells with high percent.mt. cutting cells with &gt;20% mt based on the figure above and previous analysis of processed data. 
samp.obj&lt;- subset(samp.obj, subset = percent.mt &lt; 20)
samp.obj&lt;- subset(samp.obj, subset = percent.mt &gt; 3) #because low mt cells cluster together into cluster of low quality cells downstream, can eliminate them here. this is very strict and will throw out potentially good cells from &quot;good&quot; clusters too. but those clusters have a range of mt percentage and will not be eliminated entirely like the junk clusters will.</code></pre>
<pre class="r"><code>head(samp.obj)</code></pre>
<pre><code>An object of class Seurat 
1 features across 3771 samples within 1 assay 
Active assay: RNA (1 features, 0 variable features)</code></pre>
<pre class="r"><code>VlnPlot(samp.obj, features = &quot;nCount_RNA&quot;,group.by = &quot;individual&quot;, pt.size = 0)</code></pre>
<p><img src="figure/SampleFilteringandSCT_Batch3Lane3.Rmd/unnamed-chunk-14-1.png" width="672" style="display: block; margin: auto;" /> 18858 often has higher count/higher feature count. Based on downstream analyses, lots of 18858 remained in a pluripotent state and iPSCs tend to have higher counts than other cells types.</p>
<pre class="r"><code>VlnPlot(samp.obj, features = &quot;nFeature_RNA&quot;,group.by = &quot;individual&quot;, pt.size = 0)</code></pre>
<p><img src="figure/SampleFilteringandSCT_Batch3Lane3.Rmd/unnamed-chunk-15-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>head(samp.obj)</code></pre>
<pre><code>An object of class Seurat 
1 features across 3771 samples within 1 assay 
Active assay: RNA (1 features, 0 variable features)</code></pre>
<pre class="r"><code>summary(samp.obj$nFeature_RNA)</code></pre>
<pre><code>   Min. 1st Qu.  Median    Mean 3rd Qu.    Max. 
   1606    2981    4321    4271    5424    8299 </code></pre>
<pre class="r"><code>#set a feature threshold
samp.obj&lt;- subset(samp.obj, subset = nFeature_RNA &gt; 1500) #again, very strict, may lose some &quot;good&quot; cells, but limits set to high quality cells and there are enough cells remaining for downstream analyses</code></pre>
<pre class="r"><code>head(samp.obj)</code></pre>
<pre><code>An object of class Seurat 
1 features across 3771 samples within 1 assay 
Active assay: RNA (1 features, 0 variable features)</code></pre>
<pre class="r"><code>#in seurat SCTransform vignette, they regress out MT percentage, but we don&#39;t. Reasoning is that MT percentage probably correlates with cell type for us so its still useful information for clustering.

samp.obj&lt;- suppressWarnings(SCTransform(samp.obj, verbose=F))</code></pre>
<pre class="r"><code>samp.obj&lt;- RunPCA(samp.obj, npcs= 100, verbose = F)</code></pre>
<pre class="r"><code>DimPlot(samp.obj, reduction = &quot;pca&quot;)</code></pre>
<p><img src="figure/SampleFilteringandSCT_Batch3Lane3.Rmd/unnamed-chunk-22-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>DimPlot(samp.obj, reduction = &quot;pca&quot;, group.by = &quot;individual&quot;)</code></pre>
<p><img src="figure/SampleFilteringandSCT_Batch3Lane3.Rmd/unnamed-chunk-22-2.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>VizDimLoadings(samp.obj, dims = 1:2, reduction = &quot;pca&quot;)</code></pre>
<p><img src="figure/SampleFilteringandSCT_Batch3Lane3.Rmd/unnamed-chunk-22-3.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>VizDimLoadings(samp.obj, dims = 3:4, reduction = &quot;pca&quot;)</code></pre>
<p><img src="figure/SampleFilteringandSCT_Batch3Lane3.Rmd/unnamed-chunk-22-4.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>VizDimLoadings(samp.obj, dims = 5:6, reduction = &quot;pca&quot;)</code></pre>
<p><img src="figure/SampleFilteringandSCT_Batch3Lane3.Rmd/unnamed-chunk-22-5.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>xlim &lt;- c(min(samp.obj@reductions$pca@cell.embeddings[,&#39;PC_1&#39;]),
          max(samp.obj@reductions$pca@cell.embeddings[,&#39;PC_1&#39;]))
ylim &lt;- c(min(samp.obj@reductions$pca@cell.embeddings[,&#39;PC_2&#39;]),
          max(samp.obj@reductions$pca@cell.embeddings[,&#39;PC_2&#39;]))

individuals &lt;- table(samp.obj$individual)
individuals &lt;- individuals[individuals&gt;50]
individuals &lt;- names(individuals)
for (i in individuals)
{
  print(DimPlot(samp.obj, reduction = &quot;pca&quot;, 
                cells = WhichCells(samp.obj, expression = individual == i)) +
          xlim(xlim) + ylim(ylim) + ggtitle(i))
}</code></pre>
<p><img src="figure/SampleFilteringandSCT_Batch3Lane3.Rmd/unnamed-chunk-23-1.png" width="672" style="display: block; margin: auto;" /><img src="figure/SampleFilteringandSCT_Batch3Lane3.Rmd/unnamed-chunk-23-2.png" width="672" style="display: block; margin: auto;" /><img src="figure/SampleFilteringandSCT_Batch3Lane3.Rmd/unnamed-chunk-23-3.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>#Keep all dims that explain more than 1% of variance
pva&lt;- samp.obj@reductions$pca@stdev^2/samp.obj@reductions$pca@misc$total.variance
ndim &lt;- length(which(pva&gt;=0.01))
ElbowPlot(samp.obj, ndims = ndim*2) + geom_vline(xintercept=ndim, linetype=&quot;dashed&quot;, color = &quot;red&quot;)</code></pre>
<p><img src="figure/SampleFilteringandSCT_Batch3Lane3.Rmd/unnamed-chunk-24-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>ndim</code></pre>
<pre><code>[1] 8</code></pre>
<pre class="r"><code>samp.clust&lt;- FindNeighbors(samp.obj, dims = 1:ndim, verbose = F)
samp.clust&lt;- FindClusters(samp.clust, verbose = F)</code></pre>
<pre class="r"><code>samp.clust&lt;- RunUMAP(samp.clust, dims=1:ndim, verbose = F)</code></pre>
<pre><code>Warning: The default method for RunUMAP has changed from calling Python UMAP via reticulate to the R-native UWOT using the cosine metric
To use Python UMAP via reticulate, set umap.method to &#39;umap-learn&#39; and metric to &#39;correlation&#39;
This message will be shown once per session</code></pre>
<pre class="r"><code>samp.clust&lt;- RunTSNE(samp.clust, dims=1:ndim, verbose = F)</code></pre>
<pre class="r"><code>DimPlot(samp.clust, reduction = &quot;umap&quot;)</code></pre>
<p><img src="figure/SampleFilteringandSCT_Batch3Lane3.Rmd/unnamed-chunk-27-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>DimPlot(samp.clust, reduction = &quot;umap&quot;, group.by = &quot;orig.ident&quot;)</code></pre>
<p><img src="figure/SampleFilteringandSCT_Batch3Lane3.Rmd/unnamed-chunk-27-2.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>DimPlot(samp.clust, reduction = &quot;umap&quot;, group.by = &quot;individual&quot;)</code></pre>
<p><img src="figure/SampleFilteringandSCT_Batch3Lane3.Rmd/unnamed-chunk-27-3.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>xlim &lt;- c(min(samp.clust@reductions$umap@cell.embeddings[,&#39;UMAP_1&#39;]),
          max(samp.clust@reductions$umap@cell.embeddings[,&#39;UMAP_1&#39;]))
ylim &lt;- c(min(samp.clust@reductions$umap@cell.embeddings[,&#39;UMAP_2&#39;]),
          max(samp.clust@reductions$umap@cell.embeddings[,&#39;UMAP_2&#39;]))
for (i in individuals)
{
  print(DimPlot(samp.clust, reduction = &quot;umap&quot;, 
                cells = WhichCells(samp.clust, expression = individual == i)) +
          xlim(xlim) + ylim(ylim) + ggtitle(i))
}</code></pre>
<p><img src="figure/SampleFilteringandSCT_Batch3Lane3.Rmd/unnamed-chunk-27-4.png" width="672" style="display: block; margin: auto;" /><img src="figure/SampleFilteringandSCT_Batch3Lane3.Rmd/unnamed-chunk-27-5.png" width="672" style="display: block; margin: auto;" /><img src="figure/SampleFilteringandSCT_Batch3Lane3.Rmd/unnamed-chunk-27-6.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>FeaturePlot(samp.clust, features = c(&quot;POU5F1&quot;, &quot;PAX6&quot;, &quot;HAND1&quot;, &quot;SOX17&quot;), pt.size = 0.2, ncol=3)</code></pre>
<p><img src="figure/SampleFilteringandSCT_Batch3Lane3.Rmd/unnamed-chunk-28-1.png" width="672" style="display: block; margin: auto;" /> above shows the expression of marker genes in UMAP space. POU5F1- pluripotent marker SOX17- endoderm marker HAND1- mesoderm marker PAX6- early ectoderm marker</p>
<pre class="r"><code>DimPlot(samp.clust, reduction = &quot;tsne&quot;)</code></pre>
<p><img src="figure/SampleFilteringandSCT_Batch3Lane3.Rmd/unnamed-chunk-29-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>DimPlot(samp.clust, reduction = &quot;tsne&quot;, group.by = &quot;orig.ident&quot;)</code></pre>
<p><img src="figure/SampleFilteringandSCT_Batch3Lane3.Rmd/unnamed-chunk-29-2.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>DimPlot(samp.clust, reduction = &quot;tsne&quot;, group.by = &quot;individual&quot;)</code></pre>
<p><img src="figure/SampleFilteringandSCT_Batch3Lane3.Rmd/unnamed-chunk-29-3.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>xlim &lt;- c(min(samp.clust@reductions$tsne@cell.embeddings[,&#39;tSNE_1&#39;]),
          max(samp.clust@reductions$tsne@cell.embeddings[,&#39;tSNE_1&#39;]))
ylim &lt;- c(min(samp.clust@reductions$tsne@cell.embeddings[,&#39;tSNE_2&#39;]),
          max(samp.clust@reductions$tsne@cell.embeddings[,&#39;tSNE_2&#39;]))
for (i in individuals)
{
  print(DimPlot(samp.clust, reduction = &quot;tsne&quot;, 
                cells = WhichCells(samp.clust, expression = individual == i)) +
          xlim(xlim) + ylim(ylim) + ggtitle(i))
}</code></pre>
<p><img src="figure/SampleFilteringandSCT_Batch3Lane3.Rmd/unnamed-chunk-29-4.png" width="672" style="display: block; margin: auto;" /><img src="figure/SampleFilteringandSCT_Batch3Lane3.Rmd/unnamed-chunk-29-5.png" width="672" style="display: block; margin: auto;" /><img src="figure/SampleFilteringandSCT_Batch3Lane3.Rmd/unnamed-chunk-29-6.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>samp&lt;- samp.clust@meta.data$SampleID[1]
path&lt;- here::here(&quot;output/sampleQCrds/&quot;)
saveRDS(samp.clust, file=paste0(path,samp,&#39;.seurat.rds&#39;))</code></pre>
<pre class="r"><code>FeaturePlot(samp.clust, features = &quot;nFeature_RNA&quot;)</code></pre>
<p><img src="figure/SampleFilteringandSCT_Batch3Lane3.Rmd/unnamed-chunk-31-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>FeaturePlot(samp.clust, features = &quot;nCount_RNA&quot;)</code></pre>
<p><img src="figure/SampleFilteringandSCT_Batch3Lane3.Rmd/unnamed-chunk-32-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>VlnPlot(samp.clust, features = &quot;nFeature_RNA&quot;, pt.size = 0)</code></pre>
<p><img src="figure/SampleFilteringandSCT_Batch3Lane3.Rmd/unnamed-chunk-33-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>VlnPlot(samp.clust, features = &quot;percent.mt&quot;, pt.size = 0)</code></pre>
<p><img src="figure/SampleFilteringandSCT_Batch3Lane3.Rmd/unnamed-chunk-34-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>#how many cells per cluster from each individual?
table(samp.clust@meta.data$SCT_snn_res.0.8, samp.clust@meta.data$individual)</code></pre>
<pre><code>    
     SNG-NA18511 SNG-NA18858 SNG-NA19160
  0           93         522           4
  1          164         359          49
  2           64         292          18
  3          128          21         209
  4           94           2         203
  5          183          10          58
  6           27           0         209
  7           70           5         146
  8           41           9         134
  9          157           7          11
  10          87           3          80
  11          54           1          89
  12          14           0         116
  13           9           0          29</code></pre>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 3.6.1 (2019-07-05)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Scientific Linux 7.4 (Nitrogen)

Matrix products: default
BLAS/LAPACK: /software/openblas-0.2.19-el7-x86_64/lib/libopenblas_haswellp-r0.2.19.so

locale:
[1] C

attached base packages:
[1] parallel  stats4    stats     graphics  grDevices utils     datasets 
[8] methods   base     

other attached packages:
 [1] here_0.1-11                 dplyr_1.0.0                
 [3] ggplot2_3.3.2               DropletUtils_1.6.1         
 [5] SingleCellExperiment_1.8.0  SummarizedExperiment_1.16.1
 [7] DelayedArray_0.12.3         BiocParallel_1.20.1        
 [9] matrixStats_0.56.0          Biobase_2.46.0             
[11] GenomicRanges_1.38.0        GenomeInfoDb_1.22.1        
[13] IRanges_2.20.2              S4Vectors_0.24.4           
[15] BiocGenerics_0.32.0         Matrix_1.2-18              
[17] Seurat_3.2.0                knitr_1.29                 
[19] workflowr_1.6.2            

loaded via a namespace (and not attached):
  [1] backports_1.1.8        plyr_1.8.6             igraph_1.2.5          
  [4] lazyeval_0.2.2         splines_3.6.1          listenv_0.8.0         
  [7] digest_0.6.25          htmltools_0.5.0        gdata_2.18.0          
 [10] magrittr_1.5           tensor_1.5             cluster_2.1.0         
 [13] ROCR_1.0-7             limma_3.42.2           globals_0.12.5        
 [16] R.utils_2.9.2          colorspace_1.4-1       rappdirs_0.3.1        
 [19] ggrepel_0.8.2          xfun_0.16              crayon_1.3.4          
 [22] RCurl_1.98-1.2         jsonlite_1.7.0         spatstat_1.64-1       
 [25] spatstat.data_1.4-3    survival_3.2-3         zoo_1.8-8             
 [28] ape_5.3                glue_1.4.1             polyclip_1.10-0       
 [31] gtable_0.3.0           zlibbioc_1.32.0        XVector_0.26.0        
 [34] leiden_0.3.3           Rhdf5lib_1.8.0         future.apply_1.6.0    
 [37] HDF5Array_1.14.4       abind_1.4-5            scales_1.1.1          
 [40] edgeR_3.28.1           miniUI_0.1.1.1         Rcpp_1.0.5            
 [43] viridisLite_0.3.0      xtable_1.8-4           dqrng_0.2.1           
 [46] reticulate_1.16        rsvd_1.0.3             htmlwidgets_1.5.1     
 [49] httr_1.4.2             gplots_3.0.4           RColorBrewer_1.1-2    
 [52] ellipsis_0.3.1         ica_1.0-2              farver_2.0.3          
 [55] pkgconfig_2.0.3        R.methodsS3_1.8.0      uwot_0.1.8            
 [58] deldir_0.1-28          locfit_1.5-9.4         labeling_0.3          
 [61] tidyselect_1.1.0       rlang_0.4.7            reshape2_1.4.4        
 [64] later_1.1.0.1          munsell_0.5.0          tools_3.6.1           
 [67] generics_0.0.2         ggridges_0.5.2         evaluate_0.14         
 [70] stringr_1.4.0          fastmap_1.0.1          yaml_2.2.1            
 [73] goftest_1.2-2          npsurv_0.4-0           fs_1.4.2              
 [76] fitdistrplus_1.0-14    caTools_1.18.0         purrr_0.3.4           
 [79] RANN_2.6.1             pbapply_1.4-2          future_1.18.0         
 [82] nlme_3.1-140           whisker_0.4            mime_0.9              
 [85] R.oo_1.23.0            compiler_3.6.1         plotly_4.9.2.1        
 [88] png_0.1-7              lsei_1.2-0             spatstat.utils_1.17-0 
 [91] tibble_3.0.3           stringi_1.4.6          RSpectra_0.16-0       
 [94] lattice_0.20-38        vctrs_0.3.2            pillar_1.4.6          
 [97] lifecycle_0.2.0        lmtest_0.9-37          RcppAnnoy_0.0.16      
[100] data.table_1.13.0      cowplot_1.0.0          bitops_1.0-6          
[103] irlba_2.3.3            httpuv_1.5.4           patchwork_1.0.1       
[106] R6_2.4.1               promises_1.1.1         KernSmooth_2.23-15    
[109] gridExtra_2.3          codetools_0.2-16       MASS_7.3-51.4         
[112] gtools_3.8.2           rhdf5_2.30.1           rprojroot_1.3-2       
[115] withr_2.2.0            sctransform_0.2.1      GenomeInfoDbData_1.2.2
[118] mgcv_1.8-28            grid_3.6.1             rpart_4.1-15          
[121] tidyr_1.1.0            rmarkdown_2.3          Rtsne_0.15            
[124] git2r_0.26.1           shiny_1.5.0           </code></pre>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
