---
title: "Home"
site: workflowr::wflow_site
output:
  workflowr::wflow_html:
    toc: true
    toc_depth: 1
editor_options:
  chunk_output_type: console
---

Welcome! This is a research website detailing analyses analyses performed on scRNA-seq data from human embryoid bodies, aggregates of spontaneously differentiating cells generated from induced pluripotent stem cells.

eLife article DOI: https://doi.org/10.7554/eLife.71361
Preprint on BioRxiv: https://www.biorxiv.org/content/10.1101/2021.06.16.448714v1

The overarching goals of this work are to:

1) characterize tthe diversity of cell types resulting from this particular differentiation protocol
2) Understand the contribution of biological and technical variation present in all levels of this this type of data 
  i)What is the relative contribution of individual and replicate to variation in cell composition?
  ii) What is the relative contribution of individual and replicate to variation in gene expression?
  ii) What is the relative contribution of individual and replicate to variation in gene expression dynamics across distinct developmental trajectories?
3) Evaluate the future prospects of this model system (EBs + single cell) for identification of dynamic QTLs

Study Design:

Embryoid bodies were formed from 3 human iPSCs lines in 3 replicates. Three weeks after formation, EBs were dissociated and scRNA-seq data was collected using the 10x platform. After quality control and filtering, this dataset consists of 42,488 cells.

We later collected data from 5 additional lines in a single replicate. This data was used to evaluate the robustness of EB cell type composition among YRI lines. 

# Alignment and Preprocessing

See full alignment and preprocessing pipeline here: https://github.com/kennethabarr/HumanChimp
  
# Run EmptyDrops, Add Metadata to Seurat object from each 10x lane 

See code directory: https://github.com/KLRhodes/Embryoid_Body_Pilot_Workflowr/blob/master/code/EB.getHumanMetadata.Rmd
  
# Filtering and Normalizing each 10x lane

+ [Batch1Lane1](SampleFilteringandSCT_Batch1Lane1.html)  
+ [Batch1Lane3](SampleFilteringandSCT_Batch1Lane3.html)  
+ [Batch1Lane4](SampleFilteringandSCT_Batch1Lane4.html)  
+ [Batch1Lane5](SampleFilteringandSCT_Batch1Lane5.html)  
+ [Batch1Lane6](SampleFilteringandSCT_Batch1Lane6.html)  
+ [Batch1Lane7](SampleFilteringandSCT_Batch1Lane7.html)  
+ [Batch1Lane8](SampleFilteringandSCT_Batch1Lane8.html)  
+ [Batch1Lane9](SampleFilteringandSCT_Batch1Lane9.html)  
+ [Batch2Lane1](SampleFilteringandSCT_Batch2Lane1.html)  
+ [Batch2Lane2](SampleFilteringandSCT_Batch2Lane2.html)  
+ [Batch2Lane3](SampleFilteringandSCT_Batch2Lane3.html)  
+ [Batch2Lane4](SampleFilteringandSCT_Batch2Lane4.html)  
+ [Batch3Lane1](SampleFilteringandSCT_Batch3Lane1.html)  
+ [Batch3Lane2](SampleFilteringandSCT_Batch3Lane2.html)  
+ [Batch3Lane3](SampleFilteringandSCT_Batch3Lane3.html)  
+ [Batch3Lane4](SampleFilteringandSCT_Batch3Lane4.html)  

# Merging and integration, seurat clustering


+ [Harmony correcting for Batch, Individual](IntegrateAnalysis.afterFilter.HarmonyBatchindividual.html)  


# Differential Expression with limma (Pseudobulk)

+ [Using Cluster Resolution 0.1](Pseudobulk_Limma_Harmony.BatchIndividual_ClusterRes0.1_minPCT0.2.html)
+ [Using Cluster Resolution 0.5](Pseudobulk_Limma_Harmony.BatchIndividual_ClusterRes0.5_minPCT0.2.html)
+ [Using Cluster Resolution 0.8](Pseudobulk_Limma_Harmony.BatchIndividual_ClusterRes0.8_minPCT0.2.html)
+ [Using Cluster Resolution 1](Pseudobulk_Limma_Harmony.BatchIndividual_ClusterRes1_minPCT0.2.html)


#Reference Integration

Here, we integrate our EB data with 1) scRNA-seq data from human fetal tissues (Cao et al. 2020) and with Embryoid body cells and human embryonic stem cells from Human Cell Landscape (Han et al. 2020)

+ [Integrate Reference and Annotate EB cells](IntegrateReference_SCTregressCaoPlusScHCL.html)

We then checked that the integration and annotation procedure was robust by subsetting our EB data to only cells for a particular type and re-integrated to check that most cells were annotated the same as in the full integration.

+ [Integrate Reference w/ just Early Ectoderm EB cells](IntegrateReference_SCTregressCaoPlusScHCL_JustEarlyEcto.html)
+ [Integrate Reference w/ just Endoderm EB cells](IntegrateReference_SCTregressCaoPlusScHCL_JustEndo.html)
+ [Integrate Reference w/ just Mesoderm EB cells](IntegrateReference_SCTregressCaoPlusScHCL_JustMeso.html)
+ [Integrate Reference w/ just Early Neural Crest EB cells](IntegrateReference_SCTregressCaoPlusScHCL_JustNeuralCrest.html)
+ [Integrate Reference w/ just Early Neural EB cells](IntegrateReference_SCTregressCaoPlusScHCL_JustNeuron.html)
+ [Integrate Reference w/ just Pluripotent/undifferentiated EB cells](IntegrateReference_SCTregressCaoPlusScHCL_JustPluri.html)
+ [Check Robustness of Annotations between integrations](RefInt_ComparingFulltoPartialIntegrationAnnotations.html)
+ [Check differential expression of annotated cells](ReferenceAnn_DE.html)

#Further Exploration of cell type composition across lines
+ [Visualization of 18858 EB cell type composition](18858ClusterDelve.html)
+ [Broad cell type composition of 5 additional lines](NewDataRobustCellTypeComposition.html)
+ [Reference Integration with 5 additional lines](IntegrateReference_SCTregressCaoPlusScHCL_5newlines.html)

# Topic Modelling with FastTopics
Topic modelling Scripts can be found in the code directory. 

+ [k=6 Topics: Figures](k6topics_Explore.html)
+ [k=10 Topics: Figures](k10topics_Explore.html)
+ [Topics: Replicate vs. Individual comparison](CompiledFits_BatchvInd.html)

  
#Hierachical clustering of gene expression

+ [Pseudobulk, Using Cluster Resolution 0.1](Pseudobulk_HierarchicalClustering_Harmony.Batchindividual_ClusterRes0.1_minPCT0.2.html)
+ [Pseudobulk, Using Cluster Resolution 0.5](Pseudobulk_HierarchicalClustering_Harmony.Batchindividual_ClusterRes0.5_minPCT0.2.html)
+ [Pseudobulk, Using Cluster Resolution 0.8](Pseudobulk_HierarchicalClustering_Harmony.Batchindividual_ClusterRes0.8_minPCT0.2.html)
+ [Pseudobulk, Using Cluster Resolution 1](Pseudobulk_HierarchicalClustering_Harmony.Batchindividual_ClusterRes1_minPCT0.2.html)
+ [Single Cell, Using Cluster Resolution 0.1 and no gene filter](SingleCell_HierarchicalClustering_NoGeneFilter.html)


# Variance Partition (to determine the relative contribution of Cluster, Batch, and Individual to variation in gene expression)

+ [Pseudobulk, Using Cluster Resolution 0.1](Pseudobulk_VariancePartition_Harmony.Batchindividual_ClusterRes0.1_minPCT0.2.html)
+ [Pseudobulk, Using Cluster Resolution 0.5](Pseudobulk_VariancePartition_Harmony.Batchindividual_ClusterRes0.5_minPCT0.2.html)
+ [Pseudobulk, Using Cluster Resolution 0.8](Pseudobulk_VariancePartition_Harmony.Batchindividual_ClusterRes0.8_minPCT0.2.html)
+ [Pseudobulk, Using Cluster Resolution 1](Pseudobulk_VariancePartition_Harmony.Batchindividual_ClusterRes1_minPCT0.2.html)
+ [Pseudobulk, Using Cluster Resolution 0.1 -- BY CLUSTER](Pseudobulk_VariancePartition_Harmony.Batchindividual_ClusterRes0.1_byCluster.html)
+ Single Cell, Cluster Resolution 0.1 -- see: VarPart_SC_res0.1_SCT.Rmd
+ [Single Cell, Cluster Resolution 0.1 -- BY CLUSTER](SingleCell_VariancePartitionByCluster_Harmony.Batchindividual_ClusterRes0.1_minPCT0.2.html)
+ [Single Cell VariancePartition clean figures](VarPartPlots_res0.1_SCT.html)

#Down Sampling and Power Analysis
+ [Power Analysis](DownSamp_NoiseRatio.html)

# Trajectory Inference with Scanpy/PAGA and Identification/Exploration of Dynamic Gene Module with Split-GPM

See Josh Popp's GitHub: https://github.com/jmp448/ebpilot

#Scripts for generating clean paper figures
+ [Fig. 1 - Marker gene expression, clustering, and broad cell type composition](Fig1.html)
+ [Fig. 2 - Reference Integration with original samples (3 lines, 3 replicates)](Fig2.html)
+ [Fig. 3 - Reference Integration with 5 additional lines](Fig2.5_NewLinesSeparateRefInt.html)


#Additional Processing and Analyses that were tested but ultimately not included in the paper:

Other integrations w/ and w/out harmony:

+ [Harmony correcting for Batch](IntegrateAnalysis.afterFilter.HarmonyBatch.html)
+ [Harmony correcting for Batch, Individual, and Lane](IntegrateAnalysis.afterFilter.HarmonyBatchSampleIDindividual.html)  
+ [No Integration, Just Merged](IntegrateAnalysis.afterFilter.NOHARMONYjustmerge.html)

Cell type identification using scHCL

+ [scHCL analysis](RunscHCL_HarmonyBatchInd.html)
